lib/mapper.js

```diff
@@ -53,15 +53,20 @@ exports.identify = function(identify){
  */
 
 exports.track = function(track){
+  var customer_properties = {};
+  var userId = track.userId();
+  var email = track.email();
+
+  // at least one of userId or email is required by Klaviyo
+  if (userId) customer_properties.$id = userId;
```

Comment:
I can't find docs or examples for how the mapper should behave when encountering this kind of failure. Should it throw an error up for .mapper() to handle?